config/cloud-config/master-config-template.yaml

```diff
@@ -38,19 +38,28 @@ write_files:
     permissions: 0755
     content: |
       #! /usr/bin/bash
-      
+      CALICO_VERSION=$(/usr/bin/etcdctl get /calico/version) >/dev/null 2>&1 || CALICO_VERSION="0.12.0"
```

Comment:
I'm worried about namespacing this under `/calico` since this isn't part of the Calico etcd API and data-model, rather it is owned and used by these scripts (i.e this isn't the source of truth as for which version of Calico is installed).  I think this key should probably live elsewhere - perhaps `/calico-install/version` or something similar.